tests/translator/input/function_with_sns_event_source_all_parameters.yaml

```diff
@@ -1,27 +1,28 @@
-Resources:
-  MyAwesomeFunction:
-    Type: AWS::Serverless::Function
-    Properties:
-      CodeUri: s3://sam-demo-bucket/hello.zip
-      Handler: hello.handler
-      Runtime: python2.7
-
-      Events:
-        NotificationTopic:
-          Type: SNS
-          Properties:
-            Topic: topicArn
-            Region: region
-            FilterPolicy:
-              store:
-              - example_corp
-              event:
-              - anything-but: order_cancelled
-              customer_interests:
-              - rugby
-              - football
-              - baseball
-              price_usd:
-              - numeric:
-                - '>='
-                - 100
+Resources:

```

Comment:
What changed in this file?

Reply:
I think this is eol git change, I made this commit from gitpod. I'm not sure how to resolve it. Ill update if I figure out.

The only change in translator input tests is adding the following to the SNS Event properties:

`FilterPolicyScope: MessageAttributes`

Reply:
fixed, I made the commit with git config change

`git config --global core.autocrlf false`

Reply:
Nice! We should be able to fix it at the source by ensuring it uses `\n` for line endings.